cmd/finch/virtual_machine_remove.go

```diff
@@ -34,24 +36,24 @@ func newRemoveVMAction(creator command.LimaCmdCreator, logger flog.Logger) *remo
 }
 
 func (rva *removeVMAction) runAdapter(cmd *cobra.Command, args []string) error {
-	return rva.run()
-}
-
-func (rva *removeVMAction) run() error {
-	err := rva.assertVMIsStopped(rva.creator, rva.logger)
+	force, err := cmd.Flags().GetBool("force")
 	if err != nil {
 		return err
 	}
+	return rva.run(force)
+}
 
-	limaCmd := rva.creator.CreateWithoutStdio("remove", limaInstanceName)
-	rva.logger.Info("Removing existing Finch virtual machine...")
-	logs, err := limaCmd.CombinedOutput()
+func (rva *removeVMAction) run(force bool) error {
+	if force {
+		return rva.removeVM(force)
+	}
+
+	err := rva.assertVMIsStopped(rva.creator, rva.logger)
 	if err != nil {
-		rva.logger.Errorf("Finch virtual machine failed to remove, debug logs: %s", logs)
 		return err
 	}
-	rva.logger.Info("Finch virtual machine removed successfully")
-	return nil
+
+	return rva.removeVM(force)
```

Comment:
Seeing this called with the force argument twice (even though the first time should handle the "forced" use case (since we just return on the first invocation) kinda makes me thing that we should just have two different functions, instead of overloading `rva.removeVM`. Maybe just `rva.removeVM` and `rva.forceRemoveVM`. Another option is to just call `rva.removeVM` with a hardcoded `false` argument the second time. Same feedback for the corresponding `vm stop` command. WDYT?

Reply:
I was debating on this one when I wrote this. the reason I ended up going with this approach is that it's easier to keep feature parity between the two versions if we're not duplicating the code. (e.g. I plan to follow up with an option on `vm remove` to delete the finch persistent disk, which should work for both `--force` and regular)

I could go either way on this bc I think two separate functions is more readable overall. I think if we want to keep it this way, hardcoded `false` would be a good idea.

Reply:
I think I'm good with just hardcoding `false` on the second invocation